### ISSUE TITLE
concourse-ci: invalid values in CustomScriptForLinux section

### ISSUE THREAD

**Issue by ahmetalpbalkan** [Azure/azure-quickstart-templates#2296]
`concourse-ci/azuredeploy.json` has invalid keys under its CustomScriptForLinux's `
"properties"` section:

```json
    {
      "type": "Microsoft.Compute/virtualMachines/extensions",
      "name": "[concat(parameters('vmName'),'/initdevbox')]",
      "apiVersion": "[variables('apiVersion')]",
      "location": "[resourceGroup().location]",
      "dependsOn": [
        "[concat('Microsoft.Compute/virtualMachines/', parameters('vmName'))]"
      ],
      "properties": {
        "publisher": "Microsoft.OSTCExtensions",
        "type": "CustomScriptForLinux",
        "typeHandlerVersion": "1.2",
        "settings": {
          "fileUris": [
            "[concat(variables('githubPath'),'launch_script.sh')]"
          ],
          "commandToExecute": "[concat('sh launch_script.sh',' ',variables('githubPath'))]",
          "VNET-NAME": "[variables('virtualNetworkName')]",
          "SUBNET-NAME": "[variables('subnetNameForBosh')]",
          "SUBNET-NAME-FOR-CONCOURSE": "[variables('subnetNameForConcourse')]",
          "SUBSCRIPTION-ID": "[subscription().subscriptionId]",
          "STORAGE-ACCOUNT-NAME": "[variables('newStorageAccountName')]",
          "STORAGE-ACCESS-KEY": "[listKeys(variables('storageid'),variables('apiVersion')).key1]",
          "RESOURCE-GROUP-NAME": "[resourceGroup().name]",
          "TENANT-ID": "[parameters('tenantID')]",
          "CLIENT-ID": "[parameters('clientID')]",
          "CLIENT-SECRET": "[parameters('clientSecret')]",
          "CONCOURSE-IP": "[reference('concourse').ipAddress]",
          "username": "[parameters('adminUserName')]"
        }
      }
    }

```

These unfamiliar keys (everything except `commandToExecute` and `fileUris`)  will break with the new version of CustomScriptForLinux extension which does strict json-schema validation. 

cc: @bingosummer please fix

**Comment by ahmetalpbalkan** [comment]
same for `bosh-setup/azuredeploy.json`.

**Comment by bingosummer** [comment]
Yes. WIll fix.
I will fix `bosh-setup`, and @halimacc will fix `concource-ci`.

**Comment by ahmetalpbalkan** [comment]
@halimacc any updates?

**Comment by halimacc** [comment]
@ahmetalpbalkan  working on it, will give a fix soon.

**Comment by bingosummer** [comment]
It's fixed in #2327 and #2336. Please feel free to close it.

**Comment by ahmetalpbalkan** [comment]
@bingosummer thanks!
